tools/statvar_importer/stat_vars_map.py

```diff
@@ -0,0 +1,1227 @@
+import tools.statvar_importer.property_value_utils as pv_utils
+from tools.statvar_importer.filter_data_outliers import filter_data_svobs
+from tools.statvar_importer.mcf_diff import diff_mcf_node_pvs, fingerprint_mcf_nodes
+from tools.statvar_importer import mcf_file_util
+from tools.statvar_importer.mcf_filter import drop_existing_mcf_nodes
+from tools.statvar_importer.utils import capitalize_first_char, pvs_has_any_prop, str_from_number
+from config_map import ConfigMap
+from counters import Counters, CounterOptions
+from tools.statvar_importer.schema.schema_resolver import SchemaResolver
+from dc_api_wrapper import dc_api_is_defined_dcid
+from statvar_dcid_generator import get_statvar_dcid
+from name_to_alpha2 import USSTATE_MAP
+from tools.statvar_importer.schema.schema_generator import generate_statvar_name
+from tools.statvar_importer.schema.schema_checker import sanity_check_nodes
+from tools.statvar_importer.schema.schema_generator import generate_schema_nodes
+import file_util
+
+from absl import logging
+
+import csv
+import datetime
+import os
+import re
+import sys
```

Comment:
can we order the imports in the following block, each in alphabetical orders:

external modules
from external modules

modules within this folder
from modules within this folder

This should resolve the flags issue as well.